### ISSUE TITLE
dubious errors and false positive 404s

### ISSUE THREAD

**Issue by tomkralidis** [endymion/link-checker#11]
Running check-links against the output of https://github.com/geopython/pycsw/tree/website.  Workflow:

```
$ jekyll build
$ check-links _site
Checked: _site/development/announce/1.6.0.html
Problem: _site/index.html
   Link: http://www.opengeospatial.org/resource/products/details/?pid=1104
     Response: undefined method `+' for nil:NilClass
   Link: http://www.opengeospatial.org/resource/products/details/?pid=1104
     Response: undefined method `+' for nil:NilClass
Checked: _site/community.html
   Warning: https://github.com/geopython/pycsw/issues/new
     Redirected to: https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2Fgeopython%2Fpycsw%2Fissues%2Fnew
Checked: _site/development/index.html
Checked: _site/development/announce/index.html
Checked: _site/faq.html
Problem: _site/download.html
   Link: https://build.opensuse.org/package/show?package=python-pycsw&project=Application%3AGeo
     Response: #<Net::HTTPNotFound:0x86280ec>
   Link: https://build.opensuse.org/project/show?project=Application%3AGeo
     Response: #<Net::HTTPNotFound:0x867d9c0>
Checked 89 links in 7 HTML files and found 4 errors.
make: *** [linkcheck] Error 1

```

The first problem looks like a code issue.

The links in the second problem are valid and return 200.
